aea/configurations/loader.py

```diff
@@ -24,7 +24,7 @@
 import os
```

Comment:
the changes in this file were not strictly needed. Happy to revert them if we don't like the result.